elements/campaign-display/components/campaign-display-root.js

```diff
@@ -131,9 +130,9 @@ CampaignDisplayRoot.propTypes = {
   campaign: PropTypes.shape({
     active: PropTypes.bool,
     clickthrough_url: PropTypes.string,
-    id: PropTypes.number.isRequired,
```

Comment:
Actually I agree. I don't think we want to gracefully fail here. If there's something wrong with the CMS that's causing this, sentry will catch it and we want the pain.